executor/common_linux.h

```diff
@@ -5215,3 +5215,64 @@ static long syz_80211_join_ibss(volatile long a0, volatile long a1, volatile lon
 }
 
 #endif
+
+#if SYZ_EXECUTOR || __NR_syz_clone || __NR_syz_clone3
+#if SYZ_EXECUTOR
+// The slow down multiplier is already taken into account.
+#define USLEEP_FORKED_CHILD (3 * syscall_timeout_ms)
```

Comment:
One of these multiplies ms by 1000, another doesn't. One is wrong?

Reply:
Yes. Fixed that.